docs/cloud/azureai/flow-example-sharing-in-organization.md

```diff
@@ -0,0 +1,79 @@
+# Flow Example Sharing in Organization
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../../how-to-guides/faq.md#stable-vs-experimental).
+:::
+
+In this document, we will walk you through the steps to share your flow in an organization scope. This needs to register your flow as a model with some specific properties in an organization registry. Once completed, the model will be shown as a flow example in workspace portal.
+
+## Share flow as a model in an organization registry
+
+### Prepare a `readme.md` file for the flow
+
+To make the flow easily understand, it is better to include an `readme.md` file for the flow in the flow folder. Here we'll use [this existing readme](https://github.com/microsoft/promptflow/blob/main/examples/flows/chat/chat-with-wikipedia/README.md) as an example, the readme file may include following sections:
+- The purpose of the flow.
+- The tools utilized in the flow.
+- The prerequisites required to execute the flow.
+- What knowledge can be gained from this flow.
+- Any other information you wish to share about your flow.
+
+### Prepare a model yaml for the flow
```

Comment:
Should we mention that the yaml file should be put in the flow directory?

Reply:
The yaml file should not be put into the flow directory. Since there is a path field in the yaml file which needs to be the relative path from this model file to the flow folder. So the yaml file can be placed anywhere outside the flow folder.